pom.xml

```diff
@@ -355,6 +356,7 @@
         <!--spring.version>5.1.0.BUILD-SNAPSHOT</spring.version-->
         <spring-mobile.version>2.0.0.M2</spring-mobile.version>
         <spring-webflow.version>2.5.0.RELEASE</spring-webflow.version>
+        <spring-security-ldap.version>5.3.0.RELEASE</spring-security-ldap.version>
```

Comment:
This version is managed by the spring-security bom. You don't need to define it.

Reply:
Fixed